### PR TITLE
LPD-23773 Created Trial dashboar and metrics

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Review by kevenleone — CHANGES_REQUESTED** [review]
Hi @welitonpe can you see my suggestions?

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Page.tsx [hunk 1]
```diff
@@ -55,8 +55,9 @@ const Page: React.FC<PageProps> = ({
 }) => (
 	<div className="w-100">
 		<div className="align-items-center d-flex justify-content-between">
-			<Header description={description} title={title} />
-
+			{(title || description) && (
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/Page.tsx:58]: sort

```suggestion
			{(description || title) && (
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/enums/Solutions.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/**
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/enums/Solutions.ts:1]: Why Solutions.ts? you can use a name related to the enums here

Order.ts makes more sense

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/TrialOrderTab/index.tsx [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+import {DashboardEmptyTable} from '../../../../../components/DashboardTable/DashboardEmptyTable';
+import Table from '../../../../../components/Table/Table';
+import i18n from '../../../../../i18n';
+
+type AppsTableProps = {
+	items: Order[];
+};
+
+type DropDowndItems = {
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/TrialOrderTab/index.tsx:21]:
```suggestion
type DropDownItems = {
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/TrialOrderTab/index.tsx [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+	onClick: (item?: Order) => void;
+};
+
+const ORDER_STATUS_LABEL = {
+	completed: 'success',
+	pending: 'info',
+	processing: 'secondary',
+};
+
+const consoleClouEdnv = 'https://console.liferay.cloud';
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/TrialOrderTab/index.tsx:33]:
```suggestion
const CONSOLE_CLOUD_URL = 'https://console.liferay.cloud';
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/TrialOrderTab/index.tsx [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+			window.open(order?.customFields?.['trial-virtualhost'] as string),
+	},
+	{
+		id: 2,
+		name: i18n.translate('go-to-console'),
+		onClick: () => window.open(consoleClouEdnv),
+	},
+];
+
+const TrialOrderTab: React.FC<AppsTableProps> = ({items}) => {
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/TrialOrderTab/index.tsx:49]:
```suggestion
const TrialTable: React.FC<TrialTableProps> = ({items}) => {
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/TrialOrderTab/index.tsx [hunk 6]
```diff
@@ -0,0 +1,184 @@
+/**
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/TrialOrderTab/index.tsx:1]: Rename the folder from `TrialOrderTab` to `TrialTable`

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/TrialOrderTab/index.tsx [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import {Status} from '@clayui/modal/lib/types';
+import {formatDistance} from 'date-fns';
+
+import {DashboardEmptyTable} from '../../../../../components/DashboardTable/DashboardEmptyTable';
+import Table from '../../../../../components/Table/Table';
+import i18n from '../../../../../i18n';
+
+type AppsTableProps = {
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/TrialOrderTab/index.tsx:17]:
```suggestion
type TrialTableProps = {
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/TrialOrderTab/index.tsx [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+						title: '',
+					},
+				]}
+				rows={items}
+			/>
+		</>
+	);
+};
+
+export default TrialOrderTab;
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/TrialOrderTab/index.tsx:184]:
```suggestion
export default TrialTable;
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/index.tsx [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+						growth={expired.growth}
+						growthContext={`+${expired?.lastPeriod} this week`}
+						symbol="date-time"
+						title={i18n.translate('expired')}
+						value={expired.totalCount}
+					/>
+				</div>
+
+				<div className="border d-flex flex-column justify-content-center p-6 rounded-lg">
+					<TrialOrderTab items={orderTableData?.items || []} />
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/pages/Trial/index.tsx:127]:
```suggestion
					<TrialTable items={orderTableData?.items || []} />
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Pull request closed without merging**